ext/opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -156,7 +164,7 @@ def _get_metric_descriptor(
             )
 
         if isinstance(record.aggregator, SumAggregator):
-            descriptor["metric_kind"] = MetricDescriptor.MetricKind.GAUGE
+            descriptor["metric_kind"] = MetricDescriptor.MetricKind.CUMULATIVE
```

Comment:
We discussed offline but flagging here, CUMULATIVE  causes data to be dropped for UpDownCounter instruments.